docs/continuous-delivery/gitops/secret-management/secret-management-gitops.md

```diff
@@ -0,0 +1,177 @@
+# Secret Management with GitOps
+
+GitOps primarily have git repositories in the center considered a source of truth for managing both infrastructure and application. This Infrastructure and application will require secured access to other resources of the system through secrets. Committing clear text secrets into git repositories is unacceptable even if the repositories are private to your team and organization. Teams need a secure way to handle secrets when using GitOps.
+
+There are many ways to manage secrets with GitOps and at high level can be categorized into:
+
+1. Encrypted Secrets in Git Repositories
+1. Reference to secrets stored in the external key vault
+
+## 1. Encrypted Secrets in Git Repositories
```

Comment:
Capitalization is also inconsistent here between 1 & 2. Is it consistent in this particular playbook to use numbered indexes for H2's? Fine if it is normally done - it is just unusual coming from a different playbook, so figured it was worth a mention.

Reply:
I would recommend not numbering them in this case... numbering suggests that there is an order to things, and in this case it is just two different categories without an internal order, so we should probably remove the numbering both for the titles and the list